R/cleanup_maf.R

```diff
@@ -0,0 +1,36 @@
+#' @title Cleanup MAF.
+#'
+#' @description Transform input maf columns to allow for usage of dplyr verbs.
+#'
+#' @details Transform input maf columns to allow for usage of dplyr verbs.
+#' Allowing for a stright-forward plotting workflow as well as downstream data aggregation and manipulation.
+#' This function expects a set number of columns to exist in the incoming maf in order for this to work.
+#' To view the columns, see bundled data; [GAMBLR::grande_maf].
+#'
+#' @param maf_df input MAF data frame.
+#'
+#' @return maf_df with transformed columns
+#'
+#' @import dplyr
+#' @export
+#'
+#' @examples
+#'
+#' clean_maf = cleanup_maf(maf_df = grande_maf)
```

Comment:
Since the Grande maf is now deprecated in GAMBLR.data, this example should be updated to not use this resource.